composer.json

```diff
@@ -0,0 +1,69 @@
+{
+    "name": "spryker-eco/unzer-gui",
+    "type": "library",
+    "description": "UnzerGui module",
+    "license": "MIT",
+    "require": {
+        "php": ">=7.4",
+        "spryker-eco/unzer": "dev-feature/cc-14610/dev-unzer-psp as 1.0",
+        "spryker/gui": "^3.0.0",
+        "spryker/kernel": "^3.30.0",
+        "spryker/merchant": "^3.0.0",
+        "spryker/propel-orm": "^1.6.0",
+        "spryker/store": "^1.0.0"
+    },
+    "require-dev": {
+        "codeception/module-asserts": "^1.3.0",
+        "phpstan/phpstan": "^1.2.0",
+        "spryker-shop/shop-application": "^1.0.0",
+        "spryker/code-sniffer": "*",
+        "spryker/container": "*",
+        "spryker/error-handler": "*",
+        "spryker/event": "*",
+        "spryker/log": "*",
+        "spryker/monolog": "*",
+        "spryker/propel": "*",
+        "spryker/queue": "*",
+        "spryker/silex": "*",
+        "spryker/testify": "^3.43.0"
+    },
+    "autoload": {
+        "psr-4": {
+            "SprykerEco\\": "src/SprykerEco/"
+        }
+    },
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "scripts": {
+        "test": "vendor/bin/codecept run --quiet --fail-fast --no-exit",
+        "cs-check": "phpcs -p -s --standard=vendor/spryker/code-sniffer/Spryker/ruleset.xml src/ tests/",
+        "cs-fix": "phpcbf -p --standard=vendor/spryker/code-sniffer/Spryker/ruleset.xml src/ tests/",
+        "stan": "phpstan analyse -c phpstan.neon -l 8 src/",
+        "stan-setup": "cp composer.json composer.backup && COMPOSER_MEMORY_LIMIT=-1 composer require --dev phpstan/phpstan:^0.12 && mv composer.backup composer.json",
+        "lowest": "validate-prefer-lowest",
+        "lowest-setup": "composer update --prefer-lowest --prefer-stable --prefer-dist --no-interaction && cp composer.json composer.backup && composer require --dev dereuromark/composer-prefer-lowest && mv composer.backup composer.json"
+    },
+    "repositories": [
+        {
+            "type": "git",
+            "url": "https://github.com/spryker-eco/unzer.git"
```

Comment:
Those should never be needed for public repos

Reply:
This is used only because this module is not released yet.

Reply:
And should be removed after green checks and before release